spec/text_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+describe Text do
+
+  let(:customer) { double :customer, name: 'Jongmin',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.